sql/core/src/main/scala/org/apache/spark/sql/execution/commands/ddl.scala

```diff
@@ -0,0 +1,189 @@
+/*
```

Comment:
by the way, this division of commands is kind of arbitrary. The things in `commands.scala` are *also* DDLs, so dividing that file into two smaller files this way doesn't really make sense. Instead maybe I would put all the table ones into 1 file (maybe `tableDDL.scala`), all the database ones into another (`databaseDDL.scala`) and all the function ones into yet another (`functionDDL.scala`).